src/metaswitch/clearwater/config_manager/config_access.py

```diff
@@ -845,6 +853,9 @@ def print_diff_and_syslog(config_type, config_1, config_2):
         log.info(output_str)
         print(output_str)
 
+        output_str = output_str.decode('utf-8')
+        # the output_str gets encoded as 'utf-8' during audit_log() and you
+        # cannot encode utf-8 as utf-8
```

Comment:
Would you mind moving the comment before `output_str = output_str.decode('utf-8')`? That makes it easier to read if you have comments acting as a quick introduction to following code.